server/index.js

```diff
@@ -2,12 +2,15 @@ import express from "express";
 import mongoose from "mongoose";
 const app = express();
 
-mongoose.set("debug", true);
+// mongoose.set("debug", true);
 mongoose.Promise = global.Promise;
-mongoose.connect("mongodb://localhost/checkpoint2");
-
+// mongoose.connect("mongodb://localhost/checkpoint2");
+mongoose.connect("mongodb://aaronaustintx:airhead88@ds129394.mlab.com:29394/austin-prod");
```

Comment:
This is ok for testing for now, BUT you've just exposed your database username and password. You'll need to delete and re-add your mlab addon as soon as you are done (or now) and make sure to access the URI with `mongoose.connect(process.env.MONGODB_URI)`